hail_search/test_search.py

```diff
@@ -341,6 +348,63 @@ async def test_multi_project_search(self):
             sample_data={**MULTI_PROJECT_SAMPLE_DATA, **SV_WGS_SAMPLE_DATA},
         )
 
+    async def test_both_sample_types_search(self):
+        # One family (F000011_11) in a multi-project search has identical exome and genome data.
+        expected_variants = [PROJECT_2_VARIANT, MULTI_PROJECT_VARIANT1, MULTI_PROJECT_VARIANT2, VARIANT3, VARIANT4]
+        expected_results = []
+        for variant in expected_variants:
+            v = deepcopy(variant)
+            for indiv_id, gts in v['genotypes'].items():
+                v['genotypes'][indiv_id] = [gts]
+            if 'I000015_na20885' in v['genotypes']:
+                v['genotypes']['I000015_na20885'].append({**v['genotypes']['I000015_na20885'][0], 'sampleType': 'WGS'})
+            expected_results.append(v)
+
+        await self._assert_expected_search(
+            expected_results, gene_counts=GENE_COUNTS, sample_data=MULTI_PROJECT_SAMPLE_TYPES_SAMPLE_DATA,
+        )
+
+        # Variant in family_4 is de novo in exome but maternally inherited in genome.
+        # Genome passes quality and inheritance, show genotypes for both sample types.
+        inheritance_mode = 'recessive'
+        await self._assert_expected_search(
+            [FAMILY_4_VARIANT], sample_data=FAMILY_4_SAMPLE_DATA, inheritance_mode=inheritance_mode,
+            **COMP_HET_ALL_PASS_FILTERS,
+        )
+        # Exome passes quality and inheritance, show genotypes for both sample types.
+        inheritance_mode = 'de_novo'
+        await self._assert_expected_search(
+            [FAMILY_4_VARIANT], sample_data=FAMILY_4_SAMPLE_DATA, inheritance_mode=inheritance_mode,
+        )
+
+        # Variant 1 in family_5 is inherited in exome and there is no parental data in genome.
+        # Variant 2 in family_5 is inherited in exome with low parental GQ.
+        # Variant 1 genome passes quality and inheritance (since there is no parental data), show genotypes for both sample types.
```

Comment:
You can test families with no paternal data without creating new fixtures by changing the sample_data in the request to not have the paternal sample - seqr will not consider samples with no sample data in the request body in search even if they are in the table and actually removes them in `_add_entry_sample_families`